common/src/main/java/io/netty/util/internal/NativeLibraryLoader.java

```diff
@@ -64,6 +65,9 @@
             WORKDIR = tmpdir();
             logger.debug("-Dio.netty.native.workdir: " + WORKDIR + " (io.netty.tmpdir)");
         }
+
+        DELETE_NATIVE_LIB_AFTER_LOADING = SystemPropertyUtil.getBoolean(
+                "io.netty.native.deleteLibAfterLoading", true);
```

Comment:
couldn't we build the global flag at a higher level if the low level library loader allows individual libraries to be configured?